src/styles/page.scss

```diff
@@ -0,0 +1,21 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  &__main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+  }
+
+  &__header {
+    margin-bottom: 51px;
+  }
```

Comment:
Requirements say that we need fixed paddings for the cards container (bottom the same as top).
So, it would be better if you, add not `margin-bottom` to `header`, but paddings to the cards container.

Reply:
I agree with you. Thanks for the tip!🙂 I'll fix it.

Reply:
Done)